allennlp/tests/commands/find_learning_rate_test.py

```diff
@@ -20,11 +21,7 @@
 
 
 def is_matplotlib_installed():
-    try:
-        import matplotlib  # noqa: F401 - Matplotlib is optional.
-    except:  # noqa: E722. Any exception means we don't have a working matplotlib.
-        return False
-    return True
+    return importlib.util.find_spec("matplotlib") is not None
```

Comment:
I think this is neater and avoids adding exceptions to the linters.

Reply:
It would be great in the future to isolate these changes to ultra tiny PRs that we can just approve. But sure, this is fine.

Reply:
Yeah, this started cause I had a bug after the merge, and then got fixed in another PR when I've already fixed it. But also sometimes it annoys me to open many little PRs :man_shrugging:

Reply:
I'd disagree that this is actually neater. It requires the reader to go figure out `importlib.util.find_spec` does. On the other hand, a try/except is absolutely clear and precisely the behavior we want. Bypassing a linter is not a major concern if we have good reason to do so.

Reply:
Should I change it back?

Reply:
Yes, please.